Bangazon/Views/Products/Edit.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@model Bangazon.Models.Product
```

Comment:
This file is outside scope of the ticket. May cause future merge conflicts.

Reply:
I commented out unused methods on the scaffolded controller, but I left the views "as is"